src/pages/activity/components/forms/schemas.ts

```diff
@@ -128,7 +128,7 @@ export const otherActivitySchema = baseActivitySchema.extend({
   ]),
   assetId: z.string().min(1, { message: 'Please select a security' }).optional(),
   amount: z.coerce.number().min(0).optional(),
```

Comment:
this could probably be also `amount: z.coerce.number().nonnegative().optional(),`